solr/CHANGES.txt

```diff
@@ -23,6 +23,8 @@ Improvements
 
 * SOLR-17077: When a shard rejoins leader election, leave previous election only once to save unneeded calls to Zookeeper. (Pierre Salagnac)
 
+* SOLR-16116: Apache Curator is now used to manage all Solr Zookeeper interactions. (Houston Putman, Kevin Risden, Mike Drob, David Smiley)
```

Comment:
I think my role was minor.

BTW this PR adds a dependency to Curator from solrj-zookeeper.  We should highlight solrj dependencies with more publicity than Solr server.